sway/tree/container.c

```diff
@@ -319,7 +321,13 @@ static struct sway_container *container_destroy_noreaping(
 	}
 
 	wl_signal_emit(&con->events.destroy, con);
-	ipc_event_window(con, "close");
+
+	// emit IPC event
+	if (con->type == C_VIEW) {
+		ipc_event_window(con, "close");
+	} else if (con->type == C_WORKSPACE) {
+		ipc_event_workspace(NULL, con, "empty");
```

Comment:
This is being sent when a workspace is being destroyed. Shouldn't it be sent when a workspace becomes empty?

Reply:
This seems to be the behaviour in i3. The naming confuses me too.